src/Compilers/CSharp/CscCore/project.json

```diff
@@ -1,7 +1,7 @@
 ﻿{
   "supports": {
-    "net46.app": { },
-    "dnxcore50.app": { }
+    "net46.app": {},
```

Comment:
Tag @yishaigalatzer - the package manager and the editor should probably agree on what the format should be here...